docker/docker-compose/docker-compose-3BE.yml

```diff
@@ -1,67 +1,73 @@
 version: "3.9"
 services:
-  starrocks-fe:
+  starrocks-fe-0:
     image: starrocks/fe-ubuntu:latest
-    hostname: starrocks-fe
-    container_name: starrocks-fe
-    #user: root
-    command: /opt/starrocks/fe/bin/start_fe.sh
+    hostname: starrocks-fe-0
+    command:
+      - /bin/bash
+      - -c
+      - |
+        /opt/starrocks/fe_entrypoint.sh starrocks-fe-0
+    environment:
+      - HOST_TYPE=FQDN
+      - TZ=Asia/Shanghai
     ports:
-      - 1030:8030
-      - 2020:9020
-      - 3030:9030
-#    volumes:
-#      - path/of/host/fe.conf:/opt/starrocks/fe/conf/fe.conf
-#      - path/of/host/fe/meta:/opt/starrocks/fe/meta
+      - "1030:8030"
+      - "2020:9020"
+      - "3030:9030"
+    volumes:
+       - 1fe3be_fe0_data:/opt/starrocks/fe/meta
 
-  starrocks-be1:
+  starrocks-be-0:
     image: starrocks/be-ubuntu:latest
-    #user: root
+    hostname: starrocks-be-0
     command:
       - /bin/bash
       - -c
       - |
-        sleep 15s; mysql --connect-timeout 2 -h starrocks-fe -P9030 -uroot -e "alter system add backend \"starrocks-be1:9050\";"
-        /opt/starrocks/be/bin/start_be.sh 
-
-    hostname: starrocks-be1
-    container_name: starrocks-be1
+        /opt/starrocks/be_entrypoint.sh starrocks-fe-0
+    environment:
+      - HOST_TYPE=FQDN
+      - TZ=Asia/Shanghai
     depends_on:
-      - "starrocks-fe"
-#    volumes:
-#      - path/of/host/be.conf:/opt/starrocks/be/conf/be.conf
-#      - path/of/host/starrocks-be1/storage:/opt/starrocks/be/storage
-  starrocks-be2:
+      - starrocks-fe-0
+    volumes:
+      - 1fe3be_be0_data:/opt/starrocks/be/storage
+
+  starrocks-be-1:
     image: starrocks/be-ubuntu:latest
-    #user: root
+    hostname: starrocks-be-1
     command:
       - /bin/bash
       - -c
       - |
-        sleep 15s;  mysql --connect-timeout 2 -h starrocks-fe -P9030 -uroot -e "alter system add backend \"starrocks-be2:9050\";"
-        /opt/starrocks/be/bin/start_be.sh 
-
-    hostname: starrocks-be2
-    container_name: starrocks-be2
+        /opt/starrocks/be_entrypoint.sh starrocks-fe-0
+    environment:
+      - HOST_TYPE=FQDN
+      - TZ=Asia/Shanghai
     depends_on:
-      - "starrocks-fe"
-#    volumes:
-#      - path/of/host/be.conf:/opt/starrocks/be/conf/be.conf
-#      - path/of/host/starrocks-be2/storage:/opt/starrocks/be/storage
-  starrocks-be3:
+      - starrocks-fe-0
+    volumes:
+      - 1fe3be_be1_data:/opt/starrocks/be/storage
+
+  starrocks-be-2:
     image: starrocks/be-ubuntu:latest
-    #user: root
+    hostname: starrocks-be-2
```

Comment:
although the `container_name` is not required, maybe it is good to specify the `container_name` the same as the service?

Reply:
done